plato/draw/Box.py

```diff
@@ -0,0 +1,56 @@
+import itertools
+import numpy as np
+from .Lines import Lines
+from .. import math
+from .internal import attribute_setter, ShapeDecorator, ShapeAttribute
+
+@ShapeDecorator
+class Box(Lines):
+    """Create a box primitive to draw the frame
```

Comment:
Could you follow the style of other primitives' docstrings, starting with a single phrase like `A triclinic box frame.` followed by a blank line and a paragraph of description?

https://github.com/glotzerlab/plato/blob/f48f04b33e6a6074caed3a9e82aca9934eb37755/plato/draw/Arrows2D.py#L8-L14